packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -130,7 +131,6 @@ class SelectableTableRows extends React.Component {
 
 SelectableTableRows.propTypes = propTypes;
 SelectableTableRows.defaultProps = defaultProps;
-SelectableTableRows.Row = TableRow;
```

Comment:
@bjankord @mjhenkes Should we keep this line here for passivity reasons since this is part of what is being exported?